core/README.md

```diff
@@ -14,42 +14,39 @@ Collection and Feature level.
 
 fiboa Schema defines a (limited) set of data types and a vocabulary to express
 additional constraints for these data types.
 This allows to define a clear mapping between the core specification and its encodings.
 
-- [Data types](https://github.com/fiboa/schema/blob/v0.1.0/datatypes.md)
-- [Vocabulary](https://github.com/fiboa/schema/blob/v0.1.0/README.md#vocabulary)
+- [Data types](https://github.com/fiboa/schema/blob/v0.2.0/datatypes.md)
+- [Vocabulary](https://github.com/fiboa/schema/blob/v0.2.0/README.md#vocabulary)
 
-## Collection
+## Collections
 
-Collection-level metadata must be provided in an object that contains the properties below.
-The invidiual encodings may decide to embed the collection or make it available separately.
+A Collection is a group of one or more features with a unique identifier (see property `collection`).
 
-### Properties
+Each collection must have a single set of applicable schemas.
 
-| Property Name    | Data Type      | Description |
-| ---------------- | -------------- | ----------- |
-| fiboa_version    | string         | **REQUIRED.** Version number of the fiboa specification this entity implements. |
-| fiboa_extensions | array\<string> | A list of URIs to extensions this entity implements. |
+Any property that consists of the same value across all features can be de-duplicated to the collection-level
+if more than two features are available for the collection.
+The specific location and behaviour of collection-level data is specified in the encoding-specific specifications.
 
-Generally, the version and the extensions must be uniform per Collection.
-
-Other properties are also allowed to be provided, but are not described by this specification.
-
-## Features
-
-### General Properties
+## General Properties
 
 | Property Name | Data Type      | Description |
 | ------------- | -------------- | ----------- |
-| id            | string         | **REQUIRED.** A unique identifier for the field. It must be unique within the [Collection](#collection). |
+| schemas       | array\<string> | **REQUIRED.** A list of URLs to schemas the collection implements. |
+| id            | string         | **REQUIRED.** A unique identifier for the field. It must be unique per collection, i.e. `collection` and `id` form a unique identifier. |
 | collection    | string         | The identifier of the parent collection. |
 | category      | array\<string> | A set of categories the field boundary belongs to. |
 
-**collection:** The collection identifier is usually only needed for merged datasets.
+**schemas:** The schemas the collection implements. Must be URLs to the schema YAML files.
+The schema for this specification (see above) is required to be provided.
+
+**collection:** The collection identifier is usually only needed for merged datasets and it is **required** in this case.
+A validatior can't check whether the `collection` property is required, the data providers must ensure this.
```

Comment:
How will the data providers ensure this? May be worth explaining a bit more here what a collection means / how it works.

Reply:
It's a little difficult, provider should know when this is required. But I've tried to expand it a bit.